apps/web/src/pages/DirectiveForms/DigitalServicesContractingQuestionnaire/useLabels.ts

```diff
@@ -0,0 +1,533 @@
+      id: "51Hc86",
+      description:
+        "Label for _financial authority email_ field in the _digital services contracting questionnaire_",
+    }),
+    isAuthorityInvolved: intl.formatMessage({
+      defaultMessage:
+        "Are there any other authorities involved or engaged on this contract?",
+      id: "2QDT5C",
+      description:
+        "Label for _ is authorities involved_ fieldset in the _digital services contracting questionnaire_",
+    }),
+    authoritiesInvolved: intl.formatMessage({
+      defaultMessage: "Other authorities involved / engaged on this contract",
+      id: "nfcDvX",
+      description:
+        "Label for _authorities involved_ fieldset in the _digital services contracting questionnaire_",
+    }),
+    authorityInvolvedOther: intl.formatMessage({
+      defaultMessage: "Please specify the authority",
+      id: "uIWCnb",
+      description:
+        "Label for _other authorities involved_ fieldset in the _digital services contracting questionnaire_",
+    }),
+    contractBehalfOfGc: intl.formatMessage({
+      defaultMessage:
+        "Is this contract being put in place on behalf of another Government of Canada department or agency?",
+      id: "KifUVY",
+      description:
+        "Label for _contract on behalf of gc_ fieldset in the _digital services contracting questionnaire_",
+    }),
+    contractServiceOfGc: intl.formatMessage({
+      defaultMessage:
+        "Is this contract being put in place for the purpose of service provision to another Government of Canada department or agency?",
+      id: "u42Yks",
+      description:
+        "Label for _contract of service to gc_ fieldset in the _digital services contracting questionnaire_",
+    }),
+    contractForDigitalInitiative: intl.formatMessage({
+      defaultMessage:
+        "Is this contract related to a specific digital initiative?",
+      id: "ci72ST",
+      description:
+        "Label for _contract for digital initiative_ fieldset in the _digital services contracting questionnaire_",
+    }),
+    digitalInitiativeName: intl.formatMessage({
+      defaultMessage: "Name of the digital initiative",
+      id: "6ntAxU",
+      description:
+        "Label for _name of digital initiative_ field in _authorities involved_ fieldset in the _digital services contracting questionnaire_",
+    }),
+    digitalInitiativePlanSubmitted: intl.formatMessage(
+      {
+        defaultMessage:
+          "Has a <link>digital initiative forward talent plan</link> been submitted previously for the initiative?",
+        id: "Su8FEn",
+        description:
+          "Label for _digital initiative plan submitted_ fieldset in the _digital services contracting questionnaire_",
+      },
+      {
+        link: (chunks: React.ReactNode) => {
+          const locale = getLocale(intl);
+          const url = locale === "en" ? talentPlanEn : talentPlanFr;
+          return buildExternalLink(url, chunks);
+        },
+      },
```

Comment:
I'm concerned this is going to cause issues with the `ErrorSummary` leading to nested interactive elements. Is there anyway we can look at moving supporting links to the description?

Reply:
The item being referred to:
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/10872684-09f0-49c4-931d-6043748a342e)

Reply:
Response from Rosita: Yes, lets move the link to the descriptive paragraph above

Reply:
[Move link out of label](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7771/commits/61191bfd82e231582cc8a85b92429b7bebc68b58)